src/services/MentionService.js

```diff
@@ -258,4 +259,20 @@ function MentionService($rootScope, $compile, CacheService) {
             target[0].parentNode.scrollTop = target[0].offsetTop;
         }
     }
+
+    function _repositionDropdownIfOffScreen(editor, mentionState) {
+        // wait for dropdown result to render so that we can determine whether it is on/off-screen
+        $timeout(function() {
+            var mentionElement = mentionState.mentionElement;
+            var dropdownResultElement = mentionElement.find('ul.dropdown-menu');
+            if (dropdownResultElement.children().length > 0 && !dropdownResultElement.isOnScreen()) {
+                var ckeditorBox = _getCkeditorFrame(editor).getBoundingClientRect();
+                mentionElement.css({
+                    top: ckeditorBox.top,
```

Comment:
is there a way to use the cursor position for the top... otherwise it starts to overlap with what you are typing on smaller screens

Reply:
That overlap doesn't happen on ipad and unless we also support VE on mobile devices, there wont be any issues.
To answer the question though, it is possible by adding a mousemove event listener on the whole document, constantly storing the position somewhere or dynamically add that event once "@" is selected and query for the mouse's position as the user moves the mouse, but we also have to add logic to stop the dropdown from following the mouse so that users can select on the dropdown's item. Tt would be very confusing to the users as they have to move the mouse for it to show up. It is just too complicated and not intuitive.  I think it is more practical to line the dropdown with the left toolbar instead. That would prevent the overlap from showing up even on mobile devices, but now the dropdown doesn't stay inside the editor anymore when it doesn't fit and people might find that weird? I will push a change for that and you can try it out with device simulation in google chrome's devtool. 
However, based on what I said before about devices' support, I dont think any of these is necessary though.

Reply:
i think i like the original placement better, if there's a better way we can do it in a patch along with the differentiation issue, or if there's a way to limit max results shown that can be used to control how long it is

Reply:
I will revert it then.